packages/ember-routing-views/lib/views/link.js

```diff
@@ -421,17 +412,6 @@ var LinkComponent = EmberComponent.extend({
       this.set('disabled', attrs.disabledWhen);
     }
 
-    var currentWhen = attrs['current-when'];
-
-    if (attrs.currentWhen) {
-      Ember.deprecate('Using currentWhen with {{link-to}} is deprecated in favor of `current-when`.', !attrs.currentWhen);
-      currentWhen = attrs.currentWhen;
-    }
-
-    if (currentWhen) {
```

Comment:
@cibernox even changing private APIs we like to do a check to see if there is usage in the wild.

Unfortunately it seems like there is plenty: https://github.com/search?q=currentWhen&type=Code&utf8=%E2%9C%93 so we should add a corresponding deprecation in 1.13 :-/